src/components/Square.js

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
   // For Wave 1 enable this 
   //  Component to alert a parent 
   //  component when it's clicked on.
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  const handleOnClickSquareButton = () => {
+    // if (props.value){
+    //   console.log('you cant play that square!');
+    // }else{
+      props.onClickCallback(props.id);
+    // }
+  };
+
+  return <button 
+      className='square' 
+      onClick={handleOnClickSquareButton}
+      >
+        {props.value}
+      </button>;
+};
```

Comment:
We could use object destructing so we can reference props by their key name instead.

```suggestion
const Square = ( {value, id, onClickCallback, key} ) => {
   const handleOnClickSquareButton = () => {
      onClickCallback(id);
  };

  return <button 
      className='square' 
      onClick={handleOnClickSquareButton}
      >
        {value}
      </button>;
};

```